internal/install/discovery/os_windows_validator.go

```diff
@@ -0,0 +1,62 @@
+package discovery
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/newrelic/newrelic-cli/internal/install/types"
+)
+
+type OsWindowsValidator struct{}
+
+var (
+	WindowsVersionNoLongerSupported = "This version of Windows is no longer supported"
+	WindowsNoVersionMessage         = "Failed to identified a valid version of Windows"
+)
+
+func NewOsWindowsValidator() *OsWindowsValidator {
+	validator := OsWindowsValidator{}
+
+	return &validator
+}
+
+func (v *OsWindowsValidator) Execute(m *types.DiscoveryManifest) string {
```

Comment:
Again here, it seems that the return should be an error.

Reply:
Agree for a couple reasons - returning an error (or `nil`) follows the patterns we've set forth in this repo which follow Go standards to the best of our knowledge and returning an error provides the ability to return typed errors, so we could potentially have custom error types for these error scenarios.